pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -162,6 +166,7 @@ func (p *Predictor) Reconcile(ctx context.Context, isvc *v1beta1.InferenceServic
 		var err error
 		workerObjectMeta, workerPodSpec, err = p.reconcileWorker(sRuntime, isvc, &podSpec, annotations, predictorAnnotations, isvcGeneration)
 		if err != nil {
+			isvc.Status.PropagateRawStatusWithMessages(v1beta1.PredictorComponent, v1beta1.InvalidGPUAllocation, err.Error(), corev1.ConditionFalse)
```

Comment:
question: this executes if `multiNodeEnabled` is True. Is that enough? Should there also be a rawdeployment mode check here? Not sure if now/in the future multinode can be used with serverless

Reply:
We provide the condition in the doc so I think it should be ok.